traffic_ops/client/region.go

```diff
@@ -159,3 +160,28 @@ func (to *Session) GetRegionByNamePath(name string) ([]tc.RegionName, ReqInf, er
 	}
 	return resp.Response, reqInf, nil
 }
+
+// DeleteRegion lets you DELETE a Region
```

Comment:
nit: should end with a period.

Should also mention that only one of the parameters is required.

Reply:
Added a period and explanation in 6e1617fae.